packages/terra-slide-group/tests/jest/SlideGroup.test.jsx

```diff
@@ -14,11 +14,39 @@ const slideGroup = (
 
 it('should render a default SlideGroup', () => {
   const wrapper = shallow(slideGroup);
+
+  const cssTransitions = wrapper.find('CSSTransition');
+  expect(cssTransitions.at(0).key()).toBe('1');
+  expect(cssTransitions.at(0).prop('enter')).toBe(false);
+  expect(cssTransitions.at(0).prop('enter')).toBe(false);
```

Comment:
One last question: Why are these lines being double-repeated, I shouldn't expect a change here when I call the same `expect` shouldn't I?

Reply:
Oops, yeah the other ones should be the "exit" prop. Copy/paste error I will fix now. I will change the grouping also